react-frontend/src/components/Nav/navbar.js

```diff
@@ -86,12 +86,20 @@ function NavBar() {
                 <p className="navOption">Resources</p>
               </Link>
             </li>
-            {/* <li>
-              <a href=".">Products & Services</a>
-            </li>
             <li>
-              <a href=".">Blog</a>
-            </li> */}
+              <Link
+                to="/products-services"
+                className={
+                  activePage === '/products-services' ? 'active' : 'notactive'
+                }
+              >
+                <p className="navOption">Products & Services</p>
+              </Link>
+            </li>
+            {/* 
```

Comment:
I would take this comment out entirely.  Since we aren't using a tags for the navbar links, it makes more sense to just delete the commented out place holder.